app/src/main/java/org/greenstand/android/TreeTracker/models/messages/network/responses/MessageResponse.kt

```diff
@@ -0,0 +1,17 @@
+package org.greenstand.android.TreeTracker.models.messages.network.responses
+
+import com.google.gson.annotations.SerializedName
+
+data class MessageResponse(
+    val from: String,
+    val to: String,
+    val subject: String,
+    val body: String,
+    @SerializedName("composed_at")
+    val composedAt: String,
+    @SerializedName("parent_message_id")
+    val parentMessageId: String?,
+    @SerializedName("video_link")
+    val videoLink: String?,
```

Comment:
The video link here is it a URL? Would it be viewed with YouTube or Exoplayer?

Reply:
With YouTube